label_studio/tests/test_contextlog.py

```diff
@@ -0,0 +1,25 @@
+"""This file and its contents are licensed under the Apache License 2.0. Please see the included NOTICE for copyright information and LICENSE for a copy of the license.
+"""
+import json
+
+import pytest
+import responses
+from django.test import override_settings
+
+
+@responses.activate
+@pytest.mark.django_db
+@override_settings(CONTEXTLOG_SYNC=True)
+def test_contextlog(business_client):
```

Comment:
```suggestion


@responses.activate
@pytest.mark.django_db
def test_contextlog(business_client, settings):
    settings.CONTEXTLOG_SYNC = True
```

I realized that we could leverage pytest's `settings` fixture instead to make this a little simpler